pkg/operator/controller/ingress/controller.go

```diff
@@ -230,6 +230,18 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		return reconcile.Result{Requeue: true}, nil
 	}
 
+	if ingress.Status.EndpointPublishingStrategy != nil && ingress.Status.EndpointPublishingStrategy.Type == operatorv1.LoadBalancerServiceStrategyType && ingress.Status.EndpointPublishingStrategy.LoadBalancer == nil {
+		log.Info("EndpointPublishingStrategy missing LoadBalancer")
+		// set loadbalancer to external
```

Comment:
The log message could be a little more explicit, and the comment is superfluous.
```suggestion
		log.Info("Setting default value for empty status.endpointPublishingStrategy.loadBalancer field", "ingresscontroller", ingress)
```